components/LogItem.vue

```diff
@@ -1,16 +1,12 @@
 <script setup lang="ts">
 import LazyComponent from 'v-lazy-component'
+import _ from 'underscore'
```

Comment:
We could take advantage of Tree-Shaking here
Ex: `import { compact, uniq } from 'underscore'`